models/model_predict.ipynb

```diff
@@ -0,0 +1,580 @@
+   "source": [
+    "# Text sentiment analysis with 3 classes positive, negative, neutral.\n",
+    "model_url = \"https://clarifai.com/erfan/text-classification/models/sentiment-analysis-twitter-roberta-base\"\n",
+    "file_path = \"../datasets/upload/data/text_files/positive/0_9.txt\"\n",
+    "model_prediction = Model(model_url).predict_by_filepath(file_path, input_type=\"text\")\n",
+    "print(model_prediction)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Text-To-Image\n",
+    "\n",
+    "**Image Generation** - Generate an Image from given input Prompt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Image Generation using Stable Diffusion XL\n",
+    "model_url = \"https://clarifai.com/stability-ai/stable-diffusion-2/models/stable-diffusion-xl\"\n",
+    "model_prediction = Model(model_url).predict_by_bytes(b\"A painting of a cat\", input_type=\"text\")\n",
+    "print(model_prediction)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import cv2\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "# Base64 image to numpy array\n",
+    "im_b = model_prediction.outputs[0].data.image.base64\n",
+    "image_np = np.frombuffer(im_b, np.uint8)\n",
+    "img_np = cv2.imdecode(image_np, cv2.IMREAD_COLOR)  \n",
+    "\n",
+    "plt.axis('off')\n",
+    "plt.imshow(img_np[...,::-1])"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Text-To-Audio\n",
+    "Generate Audio from given text"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_url = \"https://clarifai.com/eleven-labs/audio-generation/models/speech-synthesis\"\n",
+    "\n",
+    "model_prediction = Model(model_url).predict_by_bytes(b\"Hello, How are you doing today!\", \"text\")\n",
+    "print(model_prediction.outputs[0].data.audio.base64)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Image"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Image-To-Text\n",
+    "\n",
+    "**Image Captioning** - Generate Caption for the given input Image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_url = \"https://clarifai.com/salesforce/blip/models/general-english-image-caption-blip\"\n",
+    "image_url = \"https://s3.amazonaws.com/samples.clarifai.com/featured-models/image-captioning-statue-of-liberty.jpeg\"\n",
+    "model_prediction = Model(model_url).predict_by_url(image_url, input_type=\"image\")\n",
+    "print(model_prediction)"
```

Comment:
Same here, we can have like this `print(model_prediction.outputs[0].data.text.raw)`